rspace/src/main/scala/coop/rchain/rspace/pure/Store.scala

```diff
@@ -0,0 +1,75 @@
+package coop.rchain.rspace.pure
+
+import coop.rchain.rspace.internal.{Datum, Row, WaitingContinuation}
+
+import scala.collection.immutable.Seq
+
+/** The interface for the underlying store
+	*
+	* @tparam F a type constructor representing an effect (for example ReaderT[F, LMDBContext, ?])
```

Comment:
You could do:

```(for example `ReaderT[F, LMDBContext, ?]`)```

so that the scaladoc gets formatted better.

Reply:
Thanks, forgot about it :(